### PR TITLE
[BD-21] Link to official feature toggle annotation docs in README

### PR DESCRIPTION
**Pull request by regisb**
**Description:** Refer to official docs for documenting feature toggles.

**JIRA:** BD-21: https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943

**Reviewers:**
- [x] @robrap 

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [ ] ~~Version bumped~~
- [ ] ~~Changelog record added~~
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [ ] ~~Create a tag~~
- [ ] ~~Check new version is pushed to PyPi after tag-triggered build is 
      finished.~~
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @regisb! I've created [BLENDED-603](https://openedx.atlassian.net/browse/BLENDED-603) to keep track of it in Jira.   More details are on the [BD-21](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation) project page.


When this pull request is ready, tag your edX technical lead.

File: scripts/README.rst [hunk 1]
```diff
@@ -24,22 +24,7 @@ the `code_annotations`_ tool, to annotate and summarize the definitions of
...
-    # .. toggle_description: Used with our integration with CourseTalk to display reviews for a course.
-    # .. toggle_category: course_experience
-    # .. toggle_use_cases: monitored_rollout
-    # .. toggle_creation_date: 2017-06-19
-    # .. toggle_expiration_date: ???
-    # .. toggle_warnings: We are no longer integrating with CourseTalk, so this probably should be deprecated and the code for reviews should be removed.
-    # .. toggle_tickets: DEPR-48
-    # .. toggle_status: unsupported
-    SHOW_REVIEWS_TOOL_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_reviews_tool')
+The annotation format of feature toggles is documented `here <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__.
```
Comment (robrap) [inline scripts/README.rst:27]: Please Include more of the sentence in the link text for accessibility.

**Review by robrap — COMMENTED** [review]
Thanks.

File: scripts/README.rst (hunk 1 above)
Comment (regisb) [reply scripts/README.rst:27]: :ok_hand:

File: scripts/README.rst [hunk 2]
```diff
@@ -24,22 +24,7 @@ the `code_annotations`_ tool, to annotate and summarize the definitions of
...
-    # .. toggle_description: Used with our integration with CourseTalk to display reviews for a course.
-    # .. toggle_category: course_experience
-    # .. toggle_use_cases: monitored_rollout
-    # .. toggle_creation_date: 2017-06-19
-    # .. toggle_expiration_date: ???
-    # .. toggle_warnings: We are no longer integrating with CourseTalk, so this probably should be deprecated and the code for reviews should be removed.
-    # .. toggle_tickets: DEPR-48
-    # .. toggle_status: unsupported
-    SHOW_REVIEWS_TOOL_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_reviews_tool')
+The annotation format of feature toggles is `documented in this How To <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__.
```
Comment (robrap) [inline scripts/README.rst:27]: Better, but the link still doesn’t say what the how to is.

```suggestion
See the `How To with annotation format of feature toggles <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__.
```

**Review by robrap — APPROVED** [review]

**Pull request merged**